pkg/kubectl/describe.go

```diff
@@ -475,7 +475,7 @@ func (d *NodeDescriber) Describe(namespace, name string) (string, error) {
 		if pod.Spec.Host != name {
 			continue
 		}
-		pods = append(pods, pod)
+		pods = append(pods, &pod)
```

Comment:
After we just talked about this too. I was careful not to do it in the other pieces of code that dealt with PodList.Items. Oh well. Thanks for catching.